pkgs/by-name/js/json2kdl/package.nix

```
@@ -0,0 +1,31 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+}:
+
+rustPlatform.buildRustPackage (finalAttrs: {
+  pname = "json2kdl";
+  version = "0.2.0";
+
+  src = fetchFromGitHub {
+    owner = "AgathaSorceress";
+    repo = "json2kdl";
+    rev = "v${finalAttrs.version}";
```

nit, but since I'm already requesting changes...
```suggestion
    tag = "v${finalAttrs.version}";
```